### DISCUSSION TITLE
Read Playwright configuration file from fluent wrapper

### DISCUSSION THREAD

**Discussion by apahilajani-ontra** [hdorgeval/playwright-fluent#27] [Q&A]
Hello @hdorgeval 

Thanks for putting this wonderful wrapper to Playwright.  

I am curious if there's any workaround to allow the fluent wrapper to read the config values from playwright.config.ts files?  I see you have withOptions exposed but not sure how to use it to allow the fluent wrapper to automatically capture screenshots on test failures.  The native Playwright has this config value of 'screenshot' which could be set to `only-on-failure` to do the same.  Wondering how we can achieve something similar with fluent.

It would be great to have the fluent wrapper read in by default the playwright.config files or take a command line argument to read a custom config file.

Thanks!

**Comment by hdorgeval** [comment]
@rubyrocks-code : really sorry for the delay. I will study your proposal and update the playwright-fluent api accordingly.
I try my best to publish it in version 1.57.0
Thanks !
